CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## [HEAD]
+> May 23, 2017
+
+- Expose `getCurrentIndex` method and add new `getKeysLength` method that returns history keys count ([#475])
+
```

Comment:
Please don't add to the changelog in the PR. This will cause merge conflicts if anyone else does the same.